extensions/src/ACRE2TS/TS3Client.cpp

```diff
@@ -117,6 +115,28 @@ acre::Result CTS3Client::exPersistVersion( void ) {
     return acre::Result::error;
 }
 
+acre::Result CTS3Client::setSelfVariable(const char *const data) {
```

Comment:
"SelfVariable" == "ClientMetaData" ?
Doesn't seem that readable/conclusive to me.